### DISCUSSION TITLE
CaDoodle integration!

### DISCUSSION THREAD

**Discussion by madhephaestus** [kennetek/gridfinity-rebuilt-openscad#256] [Show and tell]
Hey there folks,

First off i want to say thank you all for this wonderful library!

I loved it so much I added it to CaDoodle as the standard way to integrate gridfinity boxes into my new free, open source CAD package. CaDoodle is a local install, local file version of the TinkerCAD work-flow. 

Now you can use the CaDoodle tool to configure a box using your variables, and then modify it with a simple drad and drop interface designed to be easy for kids to learn cad. I teach Tinkercad with kids in second grade, so the interface is ass easy to use and learn as possible. 

This means Gridfinity should be much more accessible to folks that dont know how to CAD! 

Check it out here if you are interested: https://cadoodlecad.com/

**Comment by elizabethburg** [comment]
Is there a CaDoodle plugin for Gridfinity? Or, are you just sharing CaDoodle with the community? 

(I'm new to GitHub, so thanks for your patience with my ignorance.)

**Comment by madhephaestus** [comment]
Yes, there is a Gridfinity plugin within CaDoodle, and that plugin uses this repository to supply the bin shapes.

**Reply by elizabethburg** [reply]
Awesome! Do I need to download anything? Can you point me in the right direction?

**Comment by madhephaestus** [comment]
Install CaDoodle and run it. On the right side there is the shapes pallet, with a drop-down menu at the top (default value is BasicShapes) Select from the drop down OpenSCAD Plugins, and click on the first item to bring it into your model. It will have the missing plugin icon at first. It will ask you if you want to download OpenSCAD plugin, select yes and it will set up the specific version of openscad needed for this library in the background. when its don, your blank gridfinity bin will have been created in the 3d view. you can drag-and-drop shapes on top to make your custom bin.
